libs/apnos/apnos.py

```diff
@@ -0,0 +1,140 @@
+import paramiko
+
+
+class APNOS:
+
+    def __init__(self, credentials=None):
+        self.owrt_args = "--prompt root@OpenAp -s serial --log stdout --user root --passwd openwifi"
+        if credentials is None:
+            exit()
+        self.jumphost_ip = credentials['ip']  # "192.168.200.80"
+        self.jumphost_username = credentials['username']  # "lanforge"
+        self.jumphost_password = credentials['password']  # "lanforge"
+        self.jumphost_port = credentials['port']  # 22
+        self.mode = credentials['mode']  # 1 for jumphost, 0 for direct ssh
```

Comment:
If mode is direct then where the IP address of the AP?

Reply:
The data structure itself has IP, Username and password, so in mode 1, its considered as jumphost and in mode 0, its considered as AP

Reply:
I have updated this in the latest commit